### PR TITLE
Login page preceding content pages

### PR DESCRIPTION
**Pull request by washort**

### CODE REVIEW

**Comment by openjck** [comment]
Looks like tests are failing

File: distributionviewer/core/static/js/login.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  localStorage.setItem('user_email', user.email);
+  localStorage.setItem('user_token', user.token);
+  var req = new Request('/verify_google_token/', {
+    method: 'POST',
+    credentials: 'same-origin',
+    mode: 'same-origin',
+    body: new URLSearchParams('token=' + user.token)});
+  window.fetch(req).then(function (response) {
+    console.log('signed in:', user.name);
+    window.location.pathname = document.children[0].dataset.logintarget;
```
Comment (spasovski) [inline distributionviewer/core/static/js/login.js]: I'd do `document.querySelector('html').dataset...` for readability.

File: distributionviewer/core/static/js/login.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  }).catch(function (err) {
+    console.log("Login error:", err)
```
Comment (spasovski) [inline distributionviewer/core/static/js/login.js]: Stick with `'` for consistency.

**Comment by openjck** [comment]
Updated the styles of the login and logout buttons. Unfortunately we can't change the style of the login button too much. Quoting Google from [this article](https://developers.google.com/identity/sign-in/web/build-button#building_a_button_with_a_custom_graphic):

> You must follow the branding guidelines and use the appropriate colors and icons in your button.

Everything else gets an r+ from me.

**Pull request merged**